index.js

```diff
@@ -31,26 +31,30 @@ function setup(fetch) {
       }
     }
 
-    return retry(async (bail, attempt) => {
-      const {method = 'GET'} = opts;
-      const isRetry = attempt < retryOpts.retries;
-      try {
-        // this will be retried
-        const res = await fetch(url, opts);
-        debug('status %d', res.status);
-        if (res.status >= 500 && res.status < 600 && isRetry) {
-          const err = new Error(res.statusText);
-          err.code = err.status = err.statusCode = res.status;
-          err.url = url;
+    try {
+      return await retry(async (bail, attempt) => {
```

Comment:
```suggestion
      return retry(async (bail, attempt) => {
```
see https://eslint.org/docs/rules/no-return-await#top

Reply:
I think we need `await` to catch error.

Reply:
You're right 👍